docs/api/index.md

```diff
@@ -517,14 +525,6 @@ _Parameters_
 | --------- | ------------------------------------- | ----------------------- |
 | guard     | [`NavigationGuard`](#navigationguard) | navigation guard to add |
 
-#### Examples
-
-```js
-router.beforeEach(to => {
```

Comment:
let's keep the example but use `beforeResolve`

Reply:
Done